src/test/java/teammates/ui/webapi/GetFeedbackSessionLogsActionTest.java

```diff
@@ -189,4 +197,21 @@ protected void testAccessControl() {
         verifyCanAccess(submissionParams);
     }
 
+    private void validateFeedbackSessionLogOutput(
+            List<FeedbackSessionLogEntryData> outputEntries,
+            List<FeedbackSessionLogEntryAttributes> inputEntries
+    ) {
+        for (FeedbackSessionLogEntryData entryData : outputEntries) {
+            String logType = entryData.getFeedbackSessionLogType().getLabel();
+
+            assertTrue(
```

Comment:
If you're trying to assert that the lists are equal, this is not right. This here is checking whether `inputEntries` is a superset of `outputEntries`.